lib/trino-orc/src/main/java/io/trino/orc/OrcReader.java

```diff
@@ -65,7 +65,7 @@
 
 public class OrcReader
 {
-    public static final int MAX_BATCH_SIZE = 8196;
+    public static final int MAX_BATCH_SIZE = 8192;
```

Comment:
Maybe write this as `8 * 1024` for clarity

Reply:
that's a good point

Reply:
You can put it up as a separate PR, we can merge it immediately

Reply:
https://github.com/trinodb/trino/pull/16858